openstack/cronus/values.yaml

```diff
@@ -529,6 +529,85 @@ rhea_rabbitmq:
+    clientTlsAuth: VerifyClientCertIfGiven
+    # "SAP Cloud Root CA" http://aia.pki.co.sap.com/aia/SAP%20Cloud%20Root%20CA.crt
+    clientCA: |
+      -----BEGIN CERTIFICATE-----
+      MIIFZjCCA06gAwIBAgIQGHcPvmUGa79M6pM42bGFYjANBgkqhkiG9w0BAQsFADBN
+      MQswCQYDVQQGEwJERTERMA8GA1UEBwwIV2FsbGRvcmYxDzANBgNVBAoMBlNBUCBT
+      RTEaMBgGA1UEAwwRU0FQIENsb3VkIFJvb3QgQ0EwHhcNMTkwMjEzMTExOTM2WhcN
+      MzkwMjEzMTEyNjMyWjBNMQswCQYDVQQGEwJERTERMA8GA1UEBwwIV2FsbGRvcmYx
+      DzANBgNVBAoMBlNBUCBTRTEaMBgGA1UEAwwRU0FQIENsb3VkIFJvb3QgQ0EwggIi
+      MA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQChbHLXJoe/zFag6fB3IcN3d3HT
+      Y14nSkEZIuUzYs7B96GFxQi0T/2s971JFiLfB4KaCG+UcG3dLXf1H/wewq8ahArh
+      FTsu4UR71ePUQiYlk/G68EFSy2zWYAJliXJS5k0DFMIWHD1lbSjCF3gPVJSUKf+v
+      HmWD5e9vcuiPBlSCaEnSeimYRhg0ITmi3RJ4Wu7H0Xp7tDd5z4HUKuyi9XRinfvG
+      kPALiBaX01QRC51cixmo0rhVe7qsNh7WDnLNBZeA0kkxNhLKDl8J6fQHKDdDEzmZ
+      KhK5KxL5p5YIZWZ8eEdNRoYRMXR0PxmHvRanzRvSVlXSbfqxaKlORfJJ1ah1bRNt
+      o0ngAQchTghsrRuf3Qh/2Kn29IuBy4bjKR9CdNLxGrClvX/q26rUUlz6A3lbXbwJ
+      EHSRnendRfEiia+xfZD+NG2oZW0IdTXSqkCbnBnign+uxGH5ECjuLEtvtUx6i9Ae
+      xAvK2FqIuud+AchqiZBKzmQAhUjKUoACzNP2Bx2zgJOeB0BqGvf6aldG0n2hYxJF
+      8Xssc8TBlwvAqtiubP/UxJJPs+IHqU+zjm7KdP6dM2sbE+J9O3n8DzOP0SDyEmWU
+      UCwnmoPOQlq1z6fH9ghcp9bDdbh6adXM8I+SUYUcfvupOzBU7rWHxDCXld/24tpI
+      FA7FRzHwKXqMSjwtBQIDAQABo0IwQDAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/
+      BAUwAwEB/zAdBgNVHQ4EFgQUHLxmKw7KjUufjZNxqQ/KZ0ZpEyIwDQYJKoZIhvcN
+      AQELBQADggIBABdSKQsh3EfVoqplSIx6X43y2Pp+kHZLtEsRWMzgO5LhYy2/Fvel
+      eRBw/XEiB5iKuEGhxHz/Gqe0gZixw3SsHB1Q464EbGT4tPQ2UiMhiiDho9hVe6tX
+      qX1FhrhycAD1xHIxMxQP/buX9s9arFZauZrpw/Jj4tGp7aEj4hypWpO9tzjdBthy
+      5vXSviU8L2HyiQpVND/Rp+dNJmVYTiFLuULRY28QbikgFO2xp9s4RNkDBnbDeTrT
+      CKWcVsmlZLPJJQZm0n2p8CvoeAsKzIULT9YSbEEBwmeqRlmbUaoT/rUGoobSFcrP
+      jrBg66y5hA2w7S3tDH0GjMpRu16b2u0hYQocUDuMlyhrkhsO+Qtqkz1ubwHCJ8PA
+      RJw6zYl9VeBtgI5F69AEJdkAgYfvPw5DJipgVuQDSv7ezi6ZcI75939ENGjSyLVy
+      4SuP99G7DuItG008T8AYFUHAM2h/yskVyvoZ8+gZx54TC9aY9gPIKyX++4bHv5BC
+      qbEdU46N05R+AIBW2KvWozQkjhSQCbzcp6DHXLoZINI6y0WOImzXrvLUSIm4CBaj
+      6MTXInIkmitdURnmpxTxLva5Kbng/u20u5ylIQKqpcD8HWX97lLVbmbnPkbpKxo+
+      LvHPhNDM3rMsLu06agF4JTbO8ANYtWQTx0PVrZKJu+8fcIaUp7MVBIVZ
+      -----END CERTIFICATE-----
+  debug: true
+  godebug: netdns=cgo
+  service:
+    type: LoadBalancer
+    annotations:
+      disco: "true"
+      disco/record: my-ingress.region.cloud.com.
+  deployment:
+    enabled: true
+    replicas: 1
+  statefulSet:
+    enabled: false
+    replicas: 1
+    podManagementPolicy: OrderedReady
+    annotations: {}
+    labels: {}
+    headless:
+      annotations: {}
+      labels: {}
+      servicePort: 80
+  updateStrategy: RollingUpdate
+  terminationGracePeriod: 60
+  # all below can be used only with a deployment style
+  maxSurge: 1
+  maxUnavailable: 0
+  ingressAnnotations:
+    # don't handle this resource by ingress controllers
+    kubernetes.io/ingress.class: rhea
```

Comment:
This looks wrong, and something I dont think we need to add. Its been a while since I worked with nginx, but I believe this is referring to which ingress to use, which should be nginx AFAIK

Reply:
I fixed it at least to be "maillog".
That still means that - according to the cronus version - there will probably no nginx in front of the service, but I might be wrong.

Reply:
The nginx would be used by the ingress, so if there is external traffic from outside the cluster, it would go through the nginx and then to the service.

I was able to verify that the ingress class cronus has is invalid, and is effecttively "none":
<img width="668" alt="image" src="https://github.com/sapcc/helm-charts/assets/35958442/1e44e4b7-dcfd-438f-a889-8a5a6a689875">

Looking deeper, we only have 1 ingress class of `nginx` and this is the default. So effectively, cronus having this invalid one, it ends up using `nginx`

Reply:
THinking more.... we might want to test this out. That <none> then uses the default, as potentially it should show the default. This definitely feels hacky though as we set it to a class that doesn't exist. No idea why. Here are the available classes:
![Uploading image.png…]()

Reply:
I made a test and the default only kicks in if you leave the annotation off. So in this class, we are setting it to an invalid name in cronus in order to no use nginx. I am confused. But I found some chat in slack suggesting this was done because cronus only needs to cert creation. Still... I find this really confusing. Most likely you don't want this for maillog.

<img width="712" alt="image" src="https://github.com/sapcc/helm-charts/assets/35958442/614d9a77-e329-45b4-9daf-7093178734af">